init.lua

```diff
@@ -649,12 +628,7 @@ local function get_fields(player, ...)
 
 	elseif (fields.key_enter_field == "filter" or fields.search) and
 			fields.filter ~= "" then
-		local fltr = fields.filter:lower()
-		if data.filter == fltr then
-			return
-		end
-
-		data.filter = fltr
```

Comment:
Why?

Reply:
If I click to search I expect the rearch is re-done, even for the unchanged filter. Use-case is to add some items to the inventory in progressive and like to redo the change with new items.

Reply:
I set `data.filter = fields.filter:lower()` above

Reply:
For progressive move this is ok but otherwise there’s no reason to do this.